packages/medusa/src/api/routes/store/carts/update-payment-method.ts

```diff
@@ -0,0 +1,60 @@
+import { IsOptional, IsString } from "class-validator"
+import { defaultStoreCartFields, defaultStoreCartRelations } from "."
+import { CartService } from "../../../../services"
+import { validator } from "../../../../utils/validator"
+
+/**
+ * @oas [post] /store/carts/{id}
+ * operationId: PostCartsCartPaymentMethodUpdate
+ * summary: Update a Cart"
+ * description: "Updates a Cart."
+ * parameters:
+ *   - (path) id=* {string} The id of the Cart.
+ * requestBody:
+ *   content:
+ *     application/json:
+ *       schema:
+ *         properties:
```

Comment:
Include a `required` property for the provider id in the documentation as is done with the title in [admin/collections/create-collection.js](https://github.com/medusajs/medusa/blob/master/packages/medusa/src/api/routes/admin/collections/create-collection.js)

i.e.

```
/** requestBody:
 *   content:
 *     application/json:
 *       schema:
 *         required:
 *           - title
 *         properties:
 *           title:
 *             type: string
 *             description:  The title to identify the Collection by.
 *           handle:
 *             type: string
 *             description:  An optional handle to be used in slugs, if none is provided we will kebab-case the title.
 *           metadata:
 *             description: An optional set of key-value pairs to hold additional information.
 *             type: object
```